creator-node/src/services/sync/processSync.js

```diff
@@ -0,0 +1,387 @@
+const axios = require('axios')
```

Comment:
would appreciate a set of eyes on this to make sure there are no unintended deviations from the original code in `nodesync.js`